include/aikido/util/ExecutorThread.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef AIKIDO_UTIL_EXECUTORTHREAD_HPP_
+#define AIKIDO_UTIL_EXECUTORTHREAD_HPP_
+
+#include <atomic>
+#include <chrono>
+#include <functional>
+#include <thread>
+
+namespace aikido {
+namespace util {
+
+/// ExecutorThread is a wrapper of std::thread that calls a callback
+/// periodically.
+///
+/// \code
+/// ExecutorThread exec(
+///     []() { std::cout << "running\n";}, std::chrono::nanoseconds(1));
```

Comment:
This is a scary example because printing `running` will likely take longer than 1 ns. Can you change the example to something more realistic, e.g. 10 ms?